compiler/tests-common/tests/org/jetbrains/kotlin/jvm/compiler/ir/javac/AbstractIrLoadJavaUsingJavacTest.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.jvm.compiler.ir.javac
+
+import org.jetbrains.kotlin.config.CompilerConfiguration
+import org.jetbrains.kotlin.config.JVMConfigurationKeys
+import org.jetbrains.kotlin.jvm.compiler.javac.AbstractLoadJavaUsingJavacTest
+
+abstract class AbstractIrLoadJavaUsingJavacTest : AbstractLoadJavaUsingJavacTest() {
```

Comment:
I think we actually don't need this test for now. I'd prefer if we only had `AbstractIrLoadJavaTest`.

The reason is that we aren't working on javac integration (subsystem that uses javac internals to load source & binary Java classes, instead of intellij-core) for a long time already, and don't plan to in the near future. The existing `AbstractLoadJavaUsingJavacTest` is being run only to verify that we don't break what's already supported. It may well be the case that when we get to working on the javac integration again, the IR backend will already be in production, and all the non-IR tests will be removed :)

Reply:
Thanks for the context Alex. That makes perfect sense; I wasn't actually sure what the difference was between these. :)